app/services/spree_annex_cloud/reward_service.rb

```diff
@@ -10,11 +10,20 @@ def initialize(args = {})
 
     def call
```

Comment:
Actually, Service pattern just have one public method, if you think is not possible to fix that now, maybe we should register it as a tech debit.